pyidr/study_parser.py

```diff
@@ -254,7 +254,12 @@ def generate_annotation(self, component):
         def add_key_values(d, pairs):
             for key, formatter in pairs:
                 try:
-                    s.append(('%s' % key, formatter % d))
+                    value = formatter % d
+                    if '\t' in value:
```

Comment:
I think you can drop the `if ... ` since `split()` will return a list of length 1 if there's no `\t`